easybuild/easyconfigs/n/NECI/NECI-20230620-foss-2023a.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMakeCp'
+name = 'NECI'
+version = '20230620'
+_commit = '558e88c5ae6c30d0505a9badbc69111be0866ba1'
+
+homepage = 'https://github.com/ghb24/NECI_STABLE'
+description = """Standalone NECI codebase designed for FCIQMC and other stochastic quantum
+chemistry methods."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/ghb24',
+        'repo_name': 'NECI_STABLE',
+        'recursive': True,
+        'commit': _commit,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+]
+
+dependencies = [
+    ('HDF5', '1.14.0'),
+]
+
+# disable tests ending up with SEGFAULT
+preconfigopts = "sed -i '/back_spawn_excit_gen/d' '%(builddir)s/NECI_STABLE/unit_tests/CMakeLists.txt' && "
+
+# enable support for HDF5
+configopts = "-DENABLE_HDF5=ON"
+
+test_cmd = 'ctest'
+runtest = '-j'
```

Comment:
If tests are memory-hungry, disable running them in parallel (and then maybe skipping tests that segfault is not required)

Reply:
`runtest = '-j 1'` didn't help

Reply:
OK, it's just the tests that are very memory-hungry then, so it segfaults arise people will need to make sure they somehow make sure that enough memory is available...